acceptance/tests/resource/file/handle_fifo_files.rb

```diff
@@ -0,0 +1,64 @@
+test_name "should be able to handle fifo files"
+tag 'audit:high',
+    'audit:acceptance'
+confine :except, :platform => /windows/
+
+def ensure_content_to_file_manifest(file_path, ensure_value = 'present')
```

Comment:
we could make `ensure_value` not optional, since will be good to know where the method is called what is the ensure value